sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -0,0 +1,312 @@
+      device_manufacturer STRING
+    OPTIONS
+      (description = "Manufacturer of the device where the client is installed."),
+      device_model STRING
+    OPTIONS
+      (description = "Model of the device where the client is installed."),
+      os_version STRING
+    OPTIONS
+      (description = "Version of the Operating System where the client is originally installed."),
+      adjust_campaign STRING
+    OPTIONS
+      (description = "Structure parameter for the campaign name."),
+      adjust_ad_group STRING
+    OPTIONS
+      (description = "Structure parameter for the the ad group of a campaign."),
+      adjust_creative STRING
+    OPTIONS
+      (description = "Structure parameter for the creative content of a campaign."),
+      adjust_network STRING
+    OPTIONS
+      (description = "The type of source of a client installation."),
+      install_source STRING
+    OPTIONS
+      (description = "The source of a client installation."),
+      metadata STRUCT<
+        reported_first_session_ping BOOL
+        OPTIONS
+          (description = "True if the client ever reported a first_session ping."),
+          reported_metrics_ping BOOL
+        OPTIONS
+          (description = "True if the client ever reported a metrics ping."),
+          min_first_session_ping_date DATE
+        OPTIONS
+          (description = "Date of the earliest first_session ping reported."),
+          adjust_network__source_ping STRING
+        OPTIONS
+          (description = "Name of the ping that reported the first adjust_network value."),
+          install_source__source_ping STRING
+        OPTIONS
+          (description = "Name of the ping that reports the install_source value."),
+          adjust_network__source_ping_date DATE
+        OPTIONS
+          (description = "Date of the ping that reported the first adjust_network value."),
+          install_source__source_ping_date DATE
+        OPTIONS
+          (description = "Date of the ping that reported the first install_source value.")
+      >
+  )
+PARTITION BY
+  first_seen_date
+CLUSTER BY
+  channel,
+  first_reported_country,
+  adjust_network,
+  install_source
+OPTIONS
+  (
+    description = "First observations for Firefox Android clients retrieved from the earliest pings: baseline, first_session and metrics. The attributes stored in this table include the first attribution, device, OS version and ISP. This table should be accessed through the user-facing view `fenix.firefox_android_clients`. Proposal: https://docs.google.com/document/d/12bj4DhCybelqHVgOVq8KJlzgtbbUw3f68palNrv-gaM/. For more details about attribution and campaign structure see https://help.adjust.com/en/article/tracker-urls#campaign-structure-parameters."
+  );
+
+INSERT
+  `moz-fx-data-shared-prod.fenix_derived.firefox_android_clients_v1`
+WITH first_seen AS (
+  SELECT
+    client_id,
+    first_seen_date
+  FROM
+    `mozdata.org_mozilla_firefox.baseline_clients_first_seen`
```

Comment:
`mozdata.org_mozilla_firefox.baseline_clients_first_seen` contains data for the Release channel only.
Use view `mozdata.fenix.baseline_clients_first_seen` for all channels.